src/PlusDataCollection/PlusSerialLine.cxx

```diff
@@ -235,3 +235,47 @@ unsigned int SerialLine::GetNumberOfBytesAvailableForReading() const
   return 0;
 #endif
 }
+
+PlusStatus SerialLine::SetDTR(bool onOff)
+{
+#ifdef _WIN32
+  DWORD dwFunc = CLRDTR;
+  if (onOff)
+  {
+    dwFunc = SETDTR;
+  }
+  if (EscapeCommFunction(CommHandle, dwFunc))
+  {
+    return PLUS_SUCCESS;
+  }
+  else
+  {
+    return PLUS_FAIL;
+  }
+#else
+  LOG_ERROR("SerialLine::SetDTR() is only implemented on Windows");

```

Comment:
Just adding support for RTS and DTR pins wouldn't be hard, but non-windows part of the SerialLine is totally unimplemented. And I don't have a serial device which I could use for testing. After writing this code I had to ask a collaborator to check whether it is working correctly.